data/sql_updates/totals_combined.sql

```diff
@@ -1,49 +1,183 @@
+        sum(vsd.oth_loans) as all_other_loans, -- f3
+        sum(vsd.oth_loans) as other_loans_received, -- f3p
+        sum(vsd.other_disb_per) as other_disbursements,
+        sum(vsd.other_fed_op_exp_per) as other_fed_operating_expenditures,
+        sum(vsd.other_receipts) as other_fed_receipts, -- f3x
+        sum(vsd.other_receipts) as other_receipts, --f3p
+        sum(vsd.pol_pty_cmte_contb) as refunded_political_party_committee_contributions,
+        sum(vsd.pty_cmte_contb) as political_party_committee_contributions,
+        sum(vsd.shared_fed_actvy_fed_shr_per) as shared_fed_activity,
+        sum(vsd.shared_fed_actvy_nonfed_per) as allocated_federal_election_levin_share, --f3x
+        sum(vsd.shared_fed_actvy_nonfed_per) as shared_fed_activity_nonfed, --f3x
+        sum(vsd.shared_fed_op_exp_per) as shared_fed_operating_expenditures,
+        sum(vsd.shared_nonfed_op_exp_per) as shared_nonfed_operating_expenditures,
+        sum(vsd.tranf_from_nonfed_acct_per) as transfers_from_nonfed_account,
+        sum(vsd.tranf_from_nonfed_levin_per) as transfers_from_nonfed_levin,
+        sum(vsd.tranf_from_other_auth_cmte) as transfers_from_affiliated_committee, -- f3p
+        sum(vsd.tranf_from_other_auth_cmte) as transfers_from_affiliated_party, -- f3x
+        sum(vsd.tranf_from_other_auth_cmte) as transfers_from_other_authorized_committee, -- f3
+        sum(vsd.tranf_to_other_auth_cmte) as transfers_to_affiliated_committee, -- f3, f3p
+        sum(vsd.tranf_to_other_auth_cmte) as transfers_to_other_authorized_committee, -- f3x
+        sum(vsd.ttl_contb_ref) as contribution_refunds,
+        sum(vsd.ttl_contb) as contributions,
+        sum(vsd.ttl_disb) as disbursements,
+        sum(vsd.ttl_fed_elect_actvy_per) as fed_election_activity,
+        -- sum(vsd.ttl_fed_op_exp_per) as fed_operating_expenditures, -- not in debtsum
+        sum(vsd.ttl_fed_receipts_per) as fed_receipts,
+        sum(vsd.ttl_loan_repymts) as loan_repayments,
+        sum(vsd.ttl_loans) as loans_received, -- f3p
+        sum(vsd.ttl_loans) as loans, -- f3
+        sum(vsd.ttl_nonfed_tranf_per) as total_transfers,
+        sum(vsd.ttl_receipts) as receipts,
+        vsd.cmte_id as committee_id,
+        -- double check this makes sense down stream
+        -- form type can change if a committee changes this should create 2 records
+        vsd.form_tp_cd as form_type,
+        -- make it null by form type to be clear candidate's can't do independent expenditures
+        case
+            when max(last.form_type) in ('F3', 'F3P') then
+                Null
+            else
+                sum(indt_exp_per)
+        end as independent_expenditures
+    from disclosure.v_sum_and_det_sum_report vsd
+        --this was filtering results in the tests, do we really want this to be as restrictive as an inner join? -jcc
+        left join last on vsd.cmte_id = last.cmte_id and get_cycle(vsd.rpt_yr) = last.cycle
```

Comment:
I changed this from a inner join to an left join.  Was needed to get tests working.  I verified these totals as a by-product of troubleshooting the float/int issue.